src/test/javascript/spec/component/plagiarism/plagiarism-cases-review.component.spec.ts

```diff
@@ -0,0 +1,237 @@
+import { ComponentFixture, fakeAsync, TestBed, tick } from '@angular/core/testing';
+import { TranslatePipeMock } from '../../helpers/mocks/service/mock-translate.service';
+import { MockComponent, MockDirective } from 'ng-mocks';
+import { ArtemisTestModule } from '../../test.module';
+import { PlagiarismCasesReviewComponent } from 'app/course/plagiarism-cases/plagiarism-cases-review.component';
+import { PlagiarismSplitViewComponent } from 'app/exercises/shared/plagiarism/plagiarism-split-view/plagiarism-split-view.component';
+import { ActivatedRoute, convertToParamMap } from '@angular/router';
+import { PlagiarismCasesService } from 'app/course/plagiarism-cases/plagiarism-cases.service';
+import { NgModel } from '@angular/forms';
+import { MockRouterLinkDirective } from '../admin/user-management.component.spec';
```

Comment:
You'll have to adapt that import to the changes from #4343

Reply:
@MDK5 Thanks, I was just looking into that issue 😄